canonical-web-and-design/maas-ui
# Switch the Usabilla code to the 2.7 campaign

## Done
* Switch the Usabilla code to the 2.7 campaign

## QA
* Load the UI
* Check that the Feedback tab appears on the right side
* Click and check it looks like this one:
https://codepen.io/anthonydillon/pen/PowEeeE

## Fixes
Fixes https://github.com/canonical-web-and-design/MAAS-squad/issues/1714
